balancer-labs/frontend-v2
# Tokenomics - Staking home page

# Description

This change brings in the first PR for merging into the base branch for tokenomics. It is very simply in allowing the PoolsTable to get ready to show staking information

## Type of change


## How should this be tested?

Simply test that you can see the 'Stake' section under the primary pools table under 'My Investments'. The integrations and cleanups that come with, will come in future pull requests (already exist at time of this being converted to a non-draft PR)

## Visual context

https://www.loom.com/share/43e159a6a71b442f9f3492a077c3db79

## Checklist:
